tests/tests_listen.yml

```diff
@@ -29,25 +28,36 @@
     - name: Install lsof
       package:
         name: lsof
-        state: present
+        state: latest
```

Comment:
do you need it? I suggest to omit this change.

Reply:
I don't care the version of lsof...  So, reverting this. :)